File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.cs
public class Solution {
    public int MaxProfit(int[] prices) {
        if (prices.Length <= 1)
        {
            return 0;
        }
        var profit = 0;
        var min = int.MaxValue;
        for(int i = 0; i < prices.Length; i++)
        {
            if (prices[i] < min)
            {
                min = prices[i];
            }
            else
            {
                var newProfit = prices[i] - min;
                if (newProfit > profit)
                {
                    profit = newProfit;
                }
            }
        }
        return profit;
    }
}

File: Data Structures & Algorithms/is-subsequence/submission-0.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        i = 0
        for j in t:
            if i < len(s) and s[i] == j:
                i += 1

        return i == len(s)

        return True if i == len(s) - 1 else False

